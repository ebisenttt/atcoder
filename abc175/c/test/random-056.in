-536233006876936 540078636492937 760710071397670
