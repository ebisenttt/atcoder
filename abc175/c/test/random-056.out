224477064520734
